pkg/fixtures/deployments/helm/charts/values.yaml

```diff
@@ -34,8 +34,8 @@ resources:
     cpu: "1"
```

Comment:
The most recent best practice for most deployments is to have no cpu limit. I think we should adopt that. cpu is a compressible unit so having no limit there almost always leads to performance improvements

Reply:
agreed we will need to update the safeguards requirements potentially to allow this